### ISSUE TITLE
Support mintty on MINGW64

### ISSUE THREAD

**Issue by TicoDePano** [SBoudrias/Inquirer.js#621]
Mintty is installed along with Git for Windows.

Is there any chance to support mintty?

**Comment by kbrandwijk** [comment]
I don't have issues with mintty on Windows.

**Comment by sunrize531** [comment]
I can confirm that in mintty moving the cursor with arrows in prompts doesn't work at all.

**Comment by sunrize531** [comment]
So, more info on this. When script with inquirer prompt in it called directly - it works perfectly fine in mintty too. But if the same script called via `npm run` in mintty running gitbash - it's completely screwed. Arrows don't work, you can't even close it with Ctrl-C and also the output lacks colours. Probably something to do with stdout/stdin pipes in mintty.

**Comment by bitmvr** [comment]
I can second what @sunrize531 mentioned above.

If Inquirer is using `prompts`, this thread may be of value.
> https://github.com/terkelg/prompts/pull/36

**Comment by Zamiell** [comment]
using this on Git Bash for Windows also duplicates the questions, making the library unusable

e.g.

```
? Would you like to create a config file? (Y/n)    # press enter to continue
? Would you like to create it now? Yes
```

**Comment by Zamiell** [comment]
looks like this is related to #432

**Comment by Zamiell** [comment]
I noticed that Inquirer was NOT duplicating text in a simple test repository, but was duplicating text in my real app, so I did some investigating.

The issue seems to be because:
- when you type `npx foo`
- it invokes a shell script located at `C:\Users\[Username]\AppData\Roaming\npm\foo.cmd`
- which contains a line that invokes node like this:

```
endLocal & goto #_undefined_# 2>NUL || title %COMSPEC% & "%_prog%"  "%dp0%\node_modules\test\dist\main.js" %*`
```

Essentially, this command ends up up being: `"node" "path/to/foo.js"`

When I take off the quotation marks around the "node" command, the bug seems to go away, as demonstrated by the following screenshot:

![image](https://user-images.githubusercontent.com/5511220/145247765-ad6d4295-c8aa-4232-a129-b480a93a3904.png)

Pardon my French, but why the fuck is this happening? Why does putting quotation marks around node change the behavior of the resulting program? And is there some kind of fix that I can include in my JavaScript code to work around this problem?
